pupil-spa/src/app/services/user/user.service.ts

```diff
@@ -37,15 +36,14 @@ export class UserService {
         .then(data => {
           this.loggedIn = true;
           this.storageService.clear();
-          this.storageService.setItem(questionsStorageKey, data[questionsStorageKey.toString()]);
-          this.storageService.setItem(configStorageKey, data[configStorageKey.toString()]);
-          this.storageService.setItem(pupilStorageKey, data[pupilStorageKey.toString()]);
-          this.storageService.setItem(schoolStorageKey, data[schoolStorageKey.toString()]);
-          this.storageService.setItem(new AccessTokenStorageKey(),
+          this.storageService.setQuestions(data[questionsStorageKey.toString()]);
+          this.storageService.setConfig(data[configStorageKey.toString()]);
+          this.storageService.setPupil(data[pupilStorageKey.toString()]);
+          this.storageService.setSchool(data[schoolStorageKey.toString()]);
+          this.storageService.setAccessToken(
```

Comment:
would it be cleaner to just pass `data` to the storageService and let that set everything up?